caldav-webapp/src/main/webapp/vue-app/caldav/caldav-connector/caldavConnector.js

```diff
@@ -0,0 +1,99 @@
+import * as caldavConnectorService from '../js/agendaCaldavService.js';
+import * as tsdav from 'tsdav';
+import ICAL from 'ical.js';
+export default {
+  name: 'agenda.caldavCalendar',
+  description: 'agenda.caldavCalendar.description',
+  avatar: '/caldav/skin/image/caldav.png',
+  isOauth: false,
+  canConnect: true,
+  canPush: true,
+  initialized: false,
+  isSignedIn: true,
+  pushing: false,
+  rank: 40,
+  serverUrl: '',
+  connect() {
+    return new Promise((resolve, reject) => {
+      document.dispatchEvent(new CustomEvent('open-caldav-connector-settings-drawer'));
+      document.addEventListener('test-connection', (settings) => {
+        if (settings.detail) {
+          resolve('connection success');
+        } else {
+          reject('connection canceled');
+        }
+      });
+    });
+  },
+  init(connector) {
+    if (this.initialized) {
+      return;
+    }
+    this.serverUrl = connector.serverUrl;
+    this.initialized = true;
+  },
```

Comment:
Do not pass serverUrl in init, but directly in getEvents. 
Init should be empty